docker-compose-host.yml.template

```diff
@@ -1,14 +1,17 @@
-version: '2'
+version: '3.3'
 services:
   plex:
     container_name: plex
     image: plexinc/pms-docker
+    ports:
+      - "32400"
```

Comment:
Ports are not specified in host networking mode and this is the wrong way to do so.

Reply:
I missed the host mode portion.  This was out of habbit.  Removing.